Gemfile

```diff
@@ -21,12 +21,13 @@ gem 'jbuilder', '~> 2.7'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'bootstrap-sass', '3.4.1'
 # Use Active Storage variant
 # gem 'image_processing', '~> 1.2'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.4', require: false
+gem 'thor', '~> 1.1.0'
```

Comment:
may be you don't need this gem, try to remove it

Reply:
removed sir, everything still worked fine as of now.